internal/cli/show/show.go

```diff
@@ -4,19 +4,26 @@ import (
 	"github.com/alecthomas/kingpin"
 	"github.com/apex/log"
 	"github.com/ooni/probe-cli/internal/cli/root"
+	"github.com/ooni/probe-cli/internal/database"
+	"github.com/ooni/probe-cli/internal/output"
 )
 
 func init() {
 	cmd := root.Command("show", "Show a specific measurement")
 
+	msmtID := cmd.Arg("id", "the id of the measurement to show").Int64()
+
 	cmd.Action(func(_ *kingpin.ParseContext) error {
-		_, err := root.Init()
+		ctx, err := root.Init()
 		if err != nil {
 			log.WithError(err).Error("failed to initialize root context")
 			return err
 		}
-		log.Error("this function is not implemented")
-
+		msmt, err := database.GetMeasurementJSON(ctx.DB, *msmtID)
+		if err != nil {
+			log.Errorf("error: %v", err)
```

Comment:
We probably want to `return err` here

Reply:
(As a reminded, can you let me resolve conversations when I think I am satisfied with the fix, which is actually the case here? It makes my review process more streamlined and pleasant.)

Reply:
Hum, it's that then it's harder for me to track what I did fix or didn't. I suppose I can write a comment saying that it's fixed.

I generally leave them unresolved if I believe there is value in having the reviewer look at the fix to validate if it's OK. In this case the fix was very trivial and I did not consider it necessary for you to look at it.